manager/manager.go

```diff
@@ -219,7 +221,7 @@ func New(config *Config) (*Manager, error) {
 	m := &Manager{
 		config:      config,
 		listeners:   listeners,
-		caserver:    ca.NewServer(raftNode.MemoryStore(), config.SecurityConfig),
+		caserver:    ca.NewServer(raftNode.MemoryStore(), config.SecurityConfig, caReconciliationRetryInterval),
```

Comment:
`Do you think it would be better to have a method on the CA server to change the retry interval, and only use that method from tests? I'd be fine with that.`

Yes.